### PR TITLE
optimize: the Boolean.parseBoolean(String s) method should be used when converting a String to a Boolean type

### LINKED ISSUE #2084: I saw in the source code that valueOf was used when converting a string to a Boolean, which is not the case and should be done with parseBoolean
## Issue Description

Type: *bug report* or *feature request*

### Describe what you expected to happen

**I will fix the irregularities**

### How to reproduce it (as minimally and precisely as possible)

1. 
2. 
3.

### PR DESCRIPTION
**Pull request by lzf971107**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

optimize: the Boolean.parseBoolean(String s) method should be used when converting a String to a Boolean type .  fixes #2084 

## Brief changelog

1.client/src/main/java/com/alibaba/nacos/client/config/NacosConfigService.java
2.client/src/main/java/com/alibaba/nacos/client/config/impl/ServerListManager.java
3.client/src/main/java/com/alibaba/nacos/client/identify/STSConfig.java
4.client/src/main/java/com/alibaba/nacos/client/naming/utils/InitUtils.java
5.config/src/main/java/com/alibaba/nacos/config/server/service/SwitchService.java
6.config/src/main/java/com/alibaba/nacos/config/server/utils/PropertyUtil.java
7.core/src/main/java/com/alibaba/nacos/core/utils/InetUtils.java
8.naming/src/main/java/com/alibaba/nacos/naming/controllers/ServiceController.java

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [X] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

File: client/src/main/java/com/alibaba/nacos/client/config/NacosConfigService.java [hunk 1]
```diff
@@ -93,7 +93,7 @@ private void initNamespace(Properties properties) {
                 System.getProperty(SystemPropertyKeyConst.IS_USE_CLOUD_NAMESPACE_PARSING,
                     String.valueOf(Constants.DEFAULT_USE_CLOUD_NAMESPACE_PARSING)));
 
-        if (Boolean.valueOf(isUseCloudNamespaceParsing)) {
+        if (Boolean.parseBoolean(isUseCloudNamespaceParsing)) {
```
Comment (nkorange) [inline client/src/main/java/com/alibaba/nacos/client/config/NacosConfigService.java]:
```java
    /**
     * Returns a {@code Boolean} with a value represented by the
     * specified string.  The {@code Boolean} returned represents a
     * true value if the string argument is not {@code null}
     * and is equal, ignoring case, to the string {@code "true"}.
     *
     * @param   s   a string.
     * @return  the {@code Boolean} value represented by the string.
     */
    public static Boolean valueOf(String s) {
        return parseBoolean(s) ? TRUE : FALSE;
    }
```
From the JDK, these two methods seem to be identical?

Comment (lzf971107) [reply client/src/main/java/com/alibaba/nacos/client/config/NacosConfigService.java]: Yeah, but there was an extra three-wood operation

Comment (lzf971107) [reply client/src/main/java/com/alibaba/nacos/client/config/NacosConfigService.java]: You lose efficiency, even if it's just a little bit

Comment (nkorange) [reply client/src/main/java/com/alibaba/nacos/client/config/NacosConfigService.java]: OK, I think these two methods also have different return types. ```parseBoolean``` returns native type which doesn't need an extra unboxing operation.

Comment (lzf971107) [reply client/src/main/java/com/alibaba/nacos/client/config/NacosConfigService.java]: I didn't notice that. Now I've learned. Thank you

**Pull request merged**